Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -288,6 +288,20 @@ May produce spurious warnings.
 May produce spurious warnings.") {
     IsHidden = true
   };
+  public static readonly Option<bool> SuggestProofRefactoring = new("--suggest-proof-refactoring", @"
+(experimental) Emits suggestions for moving assertions into by-proofs and hiding unused function definitions.
+May produce spurious suggestions. Use with --show-hints on the CLI.") {
+    IsHidden = true
+  };
+  public static readonly Option<bool> AnalyzeProofs = new("--analyze-proofs", @"
+Uses data from to prover to suggest ways to refine the proof:
```

Comment:
Our documentation uses `verifier`, `solver` and `prover`. I'd use the term `verifier` in externally facing documentation.